packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -554,4 +554,87 @@ describeSpec('Limbo Documents:', [], () => {
         })
     );
   });
+
+  specTest(
+    'LimitToLast query from secondary results in no expected limbo doc',
+    ['multi-client'],
+    () => {
+      const limitToLast = Query.atPath(path('collection'))
+        .addOrderBy(orderBy('val', 'desc'))
+        .withLimitToLast(3);
+      const docA = doc('collection/a', 1000, { val: 11 });
+      const docB = doc('collection/b', 1000, { val: 12 });
+      const docC = doc('collection/c', 1000, { val: 13 });
+
+      const docA2 = doc('collection/a', 2000, { val: 5 });
+      const docB2 = doc('collection/b', 2000, { val: 6 });
+      const docD2 = doc('collection/d', 2000, { val: 7 });
```

Comment:
Nit: This would be easier to read if this was:

```
const docA = doc('collection/a', 1000, { val: 11 });
const docB = doc('collection/b', 1000, { val: 12 });
const docC = doc('collection/c', 1000, { val: 13 });

const docA2 = doc('collection/a', 2000, { val: 5 });
const docB2 = doc('collection/b', 2000, { val: 6 });

const docD = doc('collection/d', 2000, { val: 7 }); // No "2" and space
```